tests/tests_integration/test_api/test_data_workflows.py

```diff
@@ -432,45 +400,33 @@ def test_list_workflow_executions(
         cognite_client: CogniteClient,
         workflow_execution_list: WorkflowExecutionList,
     ) -> None:
-        workflow_ids = set(w.as_workflow_id() for w in workflow_execution_list)
-
-        assert workflow_ids, "There should be at least one workflow execution to test list with"
         listed = cognite_client.workflows.executions.list(
-            workflow_version_ids=list(workflow_ids), limit=len(workflow_execution_list)
+            workflow_version_ids=workflow_execution_list[0].as_workflow_id()
         )
 
         assert len(listed) == len(workflow_execution_list)
-        assert all(w.as_workflow_id() in workflow_ids for w in listed)
+        assert sorted(listed.dump(), key=lambda x: x["id"]) == sorted(
+            workflow_execution_list.dump(), key=lambda x: x["id"]
+        )
```

Comment:
This does what you want (you can also remove the first assert on length):
```suggestion
        unittest.TestCase().assertCountEqual(listed, workflow_execution_list)
```